### PR TITLE
Update configs ahead of deploying cromwell/cromiam 38

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: configs/caas/cromwell.conf.ctmpl [hunk 1]
```diff
@@ -158,6 +162,26 @@ system {
...
     per = 1 second
   }
+
+  # More info: https://docs.google.com/presentation/d/14VgNBDE8Don_oeOOPIWtJQC5OUrcpk3Vip5AQBBCSs4/edit
+  hog-safety {
+    # Have seen a comment referring briefly to about "4k ips per person"(???).
+    # Meanwhile, this config file sets "concurrent-job-limit = 75000".
+    # Based on the calculations in the presentation above: 75k / 4k ~= 19
+    # If you know any additional information about the IP limit, etc., please update this comment.
+    hog-factor = 19
```
Comment (cjllanwarne) [inline configs/caas/cromwell.conf.ctmpl:172]: Did we check with greenworld that this is what they want? I would guess they actually want "equivalent to no hog factors"... which would be "1"

**Review by mcovarr — APPROVED** [review]

File: configs/caas/cromwell.conf.ctmpl [hunk 2]
```diff
@@ -158,6 +162,26 @@ system {
...
+    #calculations.
+    workflow-option = "google_project"
+
+    # Interval between token logs in seconds.
+    token-log-interval-seconds = 300
+  }
+
+  # Cache file hashes within the scope of a root workflow to prevent repeatedly requesting the
+  # hashes of the same files multiple times.
+  file-hash-cache = true
```
Comment (cjllanwarne) [inline configs/caas/cromwell.conf.ctmpl:184]: Does this have any other options (like cache size?)

File: configs/caas/cromwell.conf.ctmpl [hunk 3]
```diff
@@ -412,6 +445,7 @@ database {
   profile = "slick.jdbc.MySQLProfile$"
   db {
     driver = "com.mysql.jdbc.Driver"
+    # driver = "com.mysql.cj.jdbc.Driver" <-- Switching *after* the PROD instance upgrades to 38. Til then will generate log warnings.
```
Comment (cjllanwarne) [inline configs/caas/cromwell.conf.ctmpl:448]: Is this "TODO after merging" being tracked somehow?

File: configs/caas/cromwell.conf.ctmpl (hunk 2 above)
Comment (mcovarr) [reply configs/caas/cromwell.conf.ctmpl:184]: The other parameters do not currently have knobs.

**Review by cjllanwarne — APPROVED** [review]

**Pull request merged**